packages/account/src/Components/poi/poi-form-on-signup/idv-doc-submit-on-signup/idv-doc-submit-on-signup.tsx

```diff
@@ -129,8 +129,11 @@ export const IdvDocSubmitOnSignup = ({
         }
 
         if (!document_number) {
+            const document_name = document_type.text.toLowerCase();
+
             errors.document_number =
-                localize('Please enter your document number. ') + getExampleFormat(document_type.example_format);
+                localize('Please enter your {{document_name}}. ', { document_name }) +
```

Comment:
@suisin-deriv could you please check whether this localize usage is correct?
should you be conditionally appending localize('passport'), or localize('Drivers license') and localize('social security and national insurance trust') after the string localize('Please enter your ')?

Reply:
I'm not sure whether we should do that 🤔 let me check on it 🙏